WebRoot/js/zimbraAdmin/globalconfig/view/GlobalConfigXFormView.js

```diff
@@ -1984,9 +1990,7 @@ GlobalConfigXFormView.myXFormModifier = function(xFormObject, entry) {
                                                 onActivate : "GlobalConfigXFormView.registerDeviceListener.call(this, 'block');",
                                                 enableDisableChangeEventSources : [ ZaGlobalConfig.A2_registeredDevice_Selection ],
                                                 enableDisableChecks : [ function() {
-                                                    var sel = this.getForm().getInstanceValue(
-                                                            ZaGlobalConfig.A2_registeredDevice_Selection);
-                                                    return sel && sel.length > 0;
+                                                    return GlobalConfigXFormView.shouldEnableSyncButton.call(this, [3,5])
```

Comment:
please create constants for this status values

Reply:
done